content/blog/2019-02-18-stats19.md

```diff
@@ -0,0 +1,1154 @@
+
+    ##    http://data.dft.gov.uk.s3.amazonaws.com/road-accidents-safety-data/dftRoadSafetyData_Accidents_2017.zip
+
+    ## Data already exists in data_dir, not downloading
+
+    ## Data saved at /tmp/RtmpYh76XA/dftRoadSafetyData_Accidents_2017/Acc.csv
+
+    ## Reading in:
+
+    ## /tmp/RtmpYh76XA/dftRoadSafetyData_Accidents_2017/Acc.csv
+
+``` r
+kableExtra::kable(head(crashes_2017_raw[, c(1, 4, 5, 7, 10)]))
+```
+
+<table>
+
+<thead>
+
+<tr>
+
+<th style="text-align:left;">
+
+Accident\_Index
+
+</th>
+
+<th style="text-align:right;">
+
+Longitude
+
+</th>
+
+<th style="text-align:right;">
+
+Latitude
+
+</th>
+
+<th style="text-align:right;">
+
+Accident\_Severity
+
+</th>
+
+<th style="text-align:left;">
+
+Date
+
+</th>
+
+</tr>
+
+</thead>
+
+<tbody>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010001708
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.080107
+
+</td>
+
+<td style="text-align:right;">
+
+51.65006
+
+</td>
+
+<td style="text-align:right;">
+
+1
+
+</td>
+
+<td style="text-align:left;">
+
+05/08/2017
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009342
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.173845
+
+</td>
+
+<td style="text-align:right;">
+
+51.52242
+
+</td>
+
+<td style="text-align:right;">
+
+3
+
+</td>
+
+<td style="text-align:left;">
+
+01/01/2017
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009344
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.052969
+
+</td>
+
+<td style="text-align:right;">
+
+51.51410
+
+</td>
+
+<td style="text-align:right;">
+
+3
+
+</td>
+
+<td style="text-align:left;">
+
+01/01/2017
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009348
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.060658
+
+</td>
+
+<td style="text-align:right;">
+
+51.62483
+
+</td>
+
+<td style="text-align:right;">
+
+3
+
+</td>
+
+<td style="text-align:left;">
+
+01/01/2017
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009350
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.072372
+
+</td>
+
+<td style="text-align:right;">
+
+51.57341
+
+</td>
+
+<td style="text-align:right;">
+
+2
+
+</td>
+
+<td style="text-align:left;">
+
+01/01/2017
+
+</td>
+
+</tr>
+
+<tr>
+
+<td style="text-align:left;">
+
+2017010009351
+
+</td>
+
+<td style="text-align:right;">
+
+\-0.353876
+
+</td>
+
+<td style="text-align:right;">
+
+51.43876
+
+</td>
+
+<td style="text-align:right;">
+
+3
+
+</td>
+
+<td style="text-align:left;">
+
+01/01/2017
+
+</td>
+
+</tr>
+
+</tbody>
+
+</table>
+
+Note: the severity type is not labelled (this problem affects dozens of
+columns), the column names are inconsistent, and the dates have note
+been cleaned and converted into a user-friendly date (`POSIXct`) class:
+
+``` r
+class(crashes_2017$date)
+```
+
+    ## [1] "POSIXct" "POSIXt"
+
+``` r
+class(crashes_2017_raw$Date)
+```
+
+    ## [1] "character"
+
+## Creating geographic crash data
```

Comment:
This heading is informative, while the next one "Combining the tables" is not helpful for a reader who has jumped ahead to see different parts of the example

Reply:
Or rename it...

Reply:
A renamed heading would be good if it helps the reader understand how the example is progressing (sorry that's a vague answer).

Reply:
`STATS19 is more than numbers` but settled for `The big picture: road safety`. Feel free to give it the title it deserves when you get the next commit.